hudi-common/src/main/java/org/apache/hudi/common/table/read/HoodieFileGroupReader.java

```diff
@@ -197,14 +204,19 @@ private ClosableIterator<T> makeBaseFileIterator() throws IOException {
     }
 
     StoragePathInfo baseFileStoragePathInfo = baseFile.getPathInfo();
+    // If the base file length passed in is invalid, i.e., -1,
```

Comment:
The logic is very similar with line 268, should we abstract out a utility method here?